keyboards/binepad/pixie/config.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 binepad (@binepad)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define ENCODER_DEFAULT_POS 0x3  // enable 1:1 resolution
+
+/*
+ * Wear-leveling RP2040 Driver Configuration
+ */
+
+// #define WEAR_LEVELING_RP2040_FLASH_SIZE (32 * 1024 * 1024)  // Number of bytes of flash on the board.
+// #define WEAR_LEVELING_RP2040_FLASH_BASE (flash_size-sector_size)  // The byte-wise location that the backing storage should be located.
+#define WEAR_LEVELING_LOGICAL_SIZE (2 * 1024)  // Number of bytes “exposed” to the rest of QMK and denotes the size of the usable EEPROM.
```

Comment:
This is actually smaller than the default for RP2040 (which is 4096, and 8192).

is there a specific reason for reducing the capacity?

Reply:
Reason was that since it only has 4 keys ... alas, you're right, more space for macros and less complications in the code.